docs/developing.rst

```diff
@@ -596,6 +596,80 @@ Example::
 
             toil.exportFile(outputFileID, 'file:///some/other/local/path')
 
+
+Using Docker containers in Toil
+-------------------------------
+
+Docker containers are commonly used with Toil to maintain the portability of
+workflows. The combination of Toil and Docker allows for pipelines to be fully portable
+between any platform that has both Toil and Docker installed. Docker eliminates the need for
+the user to do any other tool installation or environment setup.
+
+In order to use Docker containers with Toil, Docker must be installed
+on all workers of the cluster in order to run a workflow involving Docker containers.
+Instructions for installing Docker can be found on the `Docker`_ website.
+
+.. _Docker: https://docs.docker.com/engine/getstarted/step_one/
+
+When using Toil-based autoscaling, Docker will be automatically set up on the cluster,
```

Comment:
This is true for both cgcloud and the appliance based autoscaling. CGCloud-based autoscaling is still supported, btw.